admin/tools/docker/deployment/parallel/run.sh

```diff
@@ -63,13 +63,20 @@ else
     echo "No existing ${CONTAINER_NAME} container on any nodes"
 fi
 
+USE_JEMALLOC='/qserv/stack/stack/current/Linux64/jemalloc/master-g708711ce34/lib/libjemalloc.so'
```

Comment:
I think that is a better long term solution. We should talk to Fabrice about it. The trade off here is that this is still experimental and it's very easy to edit the run.sh file and switch between mallocs for testing. If it goes in the init.d script, we have to rebuild the container. There's probably a simple way to get the best of both worlds.